app/templates/500.html

```diff
@@ -0,0 +1,7 @@
+{% extends "base.html" %}
+
+{% block content %}
```

Comment:
"block content" does not exist anymore. Now we have two block

- anonymous when user is not logged.
- content_logged when user is logged.